### ISSUE TITLE
error warning for global <a-assets>

### ISSUE THREAD

**Issue by caseyyee** [aframevr/aframe#995]
```html
<a-assets>
  <a-mixin id="side" geometry="primitive: plane; width: 0.5; height: 2"></a-mixin>
  <a-mixin id="material" material="color: #A9F192; side: double"></a-mixin>
</a-assets>
<a-scene>
  <!-- this does not work -->
  <a-entity mixin="side material"></a-entity>

  <!-- this does -->
  <a-entity geometry="primitive: plane; width: 0.5; height: 2" material="color: #A9F192; side: double"></a-entity>
</a-scene>
````

Nothing seems to show when using mixin.

**Comment by ngokevin** [comment]
a-assets was moved to within the scene, i'll remember to mark it in the changelog

**Comment by cvan** [comment]
shouldn't this be deprecated first before changing functionality? in any event, could you at least print a helpful console warning message if this has changed?

**Comment by ngokevin** [comment]
ok

**Comment by ngokevin** [comment]
I think I will keep this as a breaking change because with the node and loading system, to have proper asset management, `<a-assets>` is required to be within `<a-scene>`. But I will add a loud error message, and we'll continue to ramp up warnings in the community to fix their versions to 0.1.2.
